#Configuation for robot odometry EKF
#
jack_disthandle_subT: "/gtec/toa/ranging"
jack_disthandle_subA: "/gtec/toa/anchors"
jack_disthandle_pubDist: "/DT5/DT"
jack_disthandle_pubGrad: "/DT5/gradient"
jackAPIName: "DT5API"
NanchorMesh: 4
Ntags: 3
TagDists: [0.0]
UWBrate: 5